app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,121 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        create_signing_key: Authentication::AuthnJwt::CreateSigningKeyInterface.new,
+        fetch_issuer_value: Authentication::AuthnJwt::FetchIssuerValue.new,
+        fetch_identity_from_token: Authentication::AuthnJwt::IdentityFromDecodedTokenProvider,
+        validate_webservice_is_whitelisted: ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
+        validate_role_can_access_webservice: ::Authentication::Security::ValidateRoleCanAccessWebservice.new,
+        validate_webservice_exists: ::Authentication::Security::ValidateWebserviceExists.new,
+        enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV),
+        validate_account_exists: ::Authentication::Security::ValidateAccountExists.new,
+        logger: Rails.logger
+      },
+      inputs: %i[authenticator_status_input]
+    ) do
+      extend(Forwardable)
+      def_delegators(:@authenticator_status_input, :authenticator_name, :account,
+                     :username, :status_webservice)
+
+      def call
+        validate_generic_status_validations
+        validate_secrets
+      end
+
+      private
+
+      def validate_generic_status_validations
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatingJwtStatusConfiguration.new)
```

Comment:
Add the logs to the functions. For example  function `validate_user_has_access_to_status_webservice` will have  @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedUserHasAccessToStatusWebservice.new) in its end

Reply:
i thought about that but it seems like too many logs